docs/status.md

```diff
@@ -84,7 +84,7 @@ one of the following tracking labels.
 | IfOp                    |         no         |        yes*        |        yes*         |          no          |        no        |
 | ImagOp                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | InfeedOp                |         no         |        yes*        |         no          |          no          |        no        |
-| IotaOp                  |         no         |        yes*        |     infeasible      |         yes          |        no        |
+| IotaOp                  |        yes         |        yes*        |     infeasible      |         yes          |       yes        |
```

Comment:
What about verification logic in the StableHLO dialect? Does it conform to the newly added spec?

Reply:
Yes, I've checked that the verifier conforms to the spec and the ODS.